### PR TITLE
Evaluate fixed costs in TwoOpt

### PR DESCRIPTION
**Pull request by leonlan**
This PR fixes the bug identified in #380, which was caused by TwoOpt not considering fixed costs.

### CODE REVIEW

File: pyvrp/tests/test_Model.py [hunk 1]
```diff
@@ -339,6 +339,31 @@ def test_model_solves_instance_with_zero_or_one_clients():
...
+    m.add_depot(x=0, y=0, tw_early=0, tw_late=40)
+
+    for idx in range(5):
+        m.add_client(x=idx, y=idx, service_duration=1, tw_early=0, tw_late=20)
+
+    for frm in m.locations:
+        for to in m.locations:
+            m.add_edge(frm, to, distance=0, duration=5)
+
+    res = m.solve(stop=MaxIterations(2000))
```
Comment (N-Wouda) [inline pyvrp/tests/test_Model.py]: Do we need this many iterations? What's the smallest amount (in, say, multiples of 100) that hangs?

**Review by N-Wouda — APPROVED** [review]
LGTM. Only comment I had was about the 2K iters, but you already handled that :)

**Comment by N-Wouda** [comment]
Let's wait for the CI and then merge it in. I'll push a new version to PyPI once it's in main.

**Comment by leonlan** [comment]
> Let's wait for the CI and then merge it in if we're hitting all cases. I'll push a new version to PyPI once it's in main.

I'll add an extra example to hit the other two cases, too.
```cpp
    if (uRoute->empty() && U->isDepot() && !n(V)->isDepot())
        deltaCost += uRoute->fixedCost();

	...

    if (!vRoute->empty() && V->isDepot() && n(U)->isDepot())
        deltaCost -= vRoute->fixedCost();
```

**Comment by leonlan** [comment]
@N-Wouda do you know why the CI doesn't want to run?

```
Current runner version: '2.308.0'
Operating System
Runner Image
Runner Image Provisioner
GITHUB_TOKEN Permissions
Secret source: Actions
Prepare workflow directory
Prepare all required actions
Getting action download info
Download action repository 'actions/checkout@v3' (SHA:f43a0e5ff2bd294095638e18286ca9a3d1956744)
Error: Can't use 'tar -xzf' extract archive file: /home/runner/work/_actions/_temp_cbec4595-d85b-4884-85db-5c7f02255017/14cb0d45-847e-4586-be74-63508db3c71b.tar.gz. return code: 2.

```

**Comment by N-Wouda** [comment]
We might need to update to checkout v4, which was release one hour ago (coincidence? I think not). In any case, I would not be surprised if there's an issue at GitHub, although the status page still claims Actions is healthy.

Pretty sure this is not us, however.

**Comment by N-Wouda** [comment]
Relevant: https://github.com/actions/checkout/issues/949.

**Comment by N-Wouda** [comment]
Maybe upgrade to checkout v4?

**Comment by leonlan** [comment]
@N-Wouda all cases are (partially) hit. You can merge it in if this is OK.

**Pull request merged**